resources/user.py

```diff
@@ -224,3 +227,23 @@ def get(self):
             } for user in users]
 
         return {"users": ret}, 200
+
+class UserInvite(Resource):
+    @jwt_required
```

Comment:
I believe this should be `@admin_required` as only admins can create users.

Reply:
Changed this.